gordo_components/dataset/base.py

```diff
@@ -73,6 +74,12 @@ def join_timeseries(
         resampled_series = []
 
         for series in series_iterable:
+            if not series.size:
```

Comment:
I guess you rely on 0 beeing false here? I must admit I prefer explicit check like `==0` if it is guaranteed to be 0. Or can it sometimes be None?

Reply:
After some discussion with Miles, I'm now catching the index error in `base.GordoDataset.get_data`, so this has been removed.